README.md

```diff
@@ -1,32 +1,239 @@
 ### [README still WIP, not contribution friendly yet]
 
-### Monitoring Server Configuration
-In `prometheus/prometheus.yml`, add hosts emitting metrics to the targets under scrape config's node job.
+Nodies Monitoring is a customizable and extensible monitoring solution for monitoring host machine metrics, container metrics, node status and node logs. 
+
+## Table of content
+
+- [Installation](#installation)
+    - [Docker Dependencies](#docker-dependencies)
+    - [Nodies Monitoring](#nodies-monitoring)
+- [Setup](#setup)
+    - [.env](#env)
+    - [chains.json](#chainsjson)
+    - [Promtail Logs](#promtail-logs)
+    - [Yaml Setup and Startup](#yaml-setup-and-startup)
+- [Customization](#customization)
+    - [Datasources](#datasources)
+    - [Dashboards](#dashboards)
+    - [Alerts](#alerts)
+- [License](#license)
+- [Links/Contact](#linkscontact)
+
+## Installation
+
+**Note:** Tested and recommended installation on Ubuntu 22.04.1 LTS host OS
+
+### Docker Dependencies
```

Comment:
Let's just list the dependencies, which then includes how to install docker. 

I believe the two dependencies are: 
- Python (which version for setup.py)
- Docker

We are missing Python